### PR TITLE
Feature/i os 128

### PR DESCRIPTION
**Pull request by gzphreak**
Add gitsubmodules for iOS-87 and iOS-93

## Description
Added gitsubmodules for 'service.bt.autosources' and 'xbmclibraryautoupdate'

This includes some gitmodules renaming and a merge, so it appears bigger than it is.

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -233,7 +233,7 @@ SKIN_DIRS=$(ESTUARY_MEDIA)
 ESTOUCHY_MEDIA=addons/skin.estouchy/media
 SKIN_DIRS+=$(ESTOUCHY_MEDIA)
 
-BTPLAY_MEDIA=addons/skin.bt.play/media
```
Comment (jjoyce-bittorrent) [inline Makefile.in]: this shouldn't have changed because .gitmodules above is addons/skin.bt.play

File: addons/skin.btplay/changelog.txt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+[B]0.8.5[/B]
```
Comment (jjoyce-bittorrent) [inline addons/skin.btplay/changelog.txt]: i don't think all of these files should have been added?

**Review by jjoyce-bittorrent — DISMISSED** [review]
maybe you need to remerge from develop? seems like there are some unrelated changes in here

File: addons/skin.btplay/changelog.txt (hunk 2 above)
Comment (gzphreak) [reply addons/skin.btplay/changelog.txt]: Agreed, However I am up to date with develop. I'll look into it.

Comment (gzphreak) [reply addons/skin.btplay/changelog.txt]: Ok, It looks like my branch squashed your submodules even though they are listed in git.

**Review by jjoyce-bittorrent — APPROVED** [review]
lgtm

**Pull request merged**